class-15.md

```diff
@@ -0,0 +1,31 @@
+# Intro to Blockchain Development
+
+### Class 15 - Exchanges 
+
+Many users don't always hold cryptocurrency inside a wallet or linked to a single address. In many instances, they purchase on an exchange, or in a system where one party takes custody of cryptocurrency for many users. 
+
+Reading & Videos:
+
+- [Hot vs. Cold Wallets](https://youtu.be/Aji_E9sw0AE)* - An Andreas Antonopoulos talk about hot and cold wallets
+
+### Intro To Exchanges
+
+While it is possible to get cryptocurrency by through the transaction fee or mining reward in a block, the majority of Bitcoin and Ethereum users do not perform mining themselves. Additionally, many miners make payments to electricity companies using fiat currencies like the renminbi or sometimes the dollar. Many miners need to sell holdings of cryptocurrency wish to sell their holdings, while many end users who wish to transact or invest purchase it. This results in a marketplace where users need to transact between traditional fiat currencies and cryptocurrencies.
+
+In many markets, exchanges exist to service these types of transactions. In the US, there is the New York Stock Exchange (NYSE) where people exchange ownership shares of companies for US Dollars and back. Currency traders also do this, when travellers wish to trade their currencies for a local currency, they can find a company to perform the exchange. At a larger scale, traders get matched up directly in a forex, where orders are opened and filled between the traders. Commodities exchanges exist as well, allowing people to buy gold, oil and agricultural goods with fiat currencies. A cryptocurrency exchange is very similar to this.
+
+Cryptocurrency exchanges are very similar to this. In a cryptocurrency exchange, a user exchanges fiat currencies or cryptocurrencies for fiat currencies or cryptocurrencies held by other users. They can be implemented in a number of ways however, some very different from a standard forex. Most cryptocurrency exchanges can be broken down into centralized or decentralized/peer-to-peer.
```

Comment:
`Cryptocurrency exchanges are very similar to this.` was repeated just before this paragraph. We can remove this.

`In a cryptocurrency exchange, a user exchanges fiat currencies or cryptocurrencies for fiat currencies or cryptocurrencies held by other users.`
This makes no sense. "In a cryptocurrency exchange, a user can choose to exchange fiat currency for cryptocurrency held by other users (i.e. USD for BTC), or their own cryptocurrency for fiat currency (i.e. ETH for USD)."